calculator.js

```diff
@@ -0,0 +1,57 @@
+var finished = false;
+do{
+  var calculator_type = prompt("Enter 'basic' , 'advanced' or 'BMI' ");
+  if(calculator_type == "basic"){
+    // get the user inputs
+    var value1 = prompt("Enter first value");
+    var value2 = prompt("Enter the second value");
+    var choice = prompt("Choose your action (a)dd (s)ubtract (m)ultiply (d)ivide (mo)dulus") || "a";
+
+    switch(choice){
+      case "a":
+        alert(value1 + value2);
```

Comment:
These alerts do work, but they are a bit abrupt and don't make for a very nice user experience. I would at least alert something like "The result is: " and then the result.